spec/features/restaurants/visitor_tries_update_restaurant_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature 'prevent unuathorized updates to restaurant' do
+  context 'as a visitor' do
+    scenario 'I cannot save a restaurant' do
+      visit new_restaurant_path
+      expect(page).to have_content("Log in")
+    end
+
+    scenario 'I cannot edit a restaurant' do
+      visit edit_restaurant_path("1")
+      expect(page).to have_content("Log in")
+    end
+
+    scenario 'I cannot delete a restaurant' do
+      visit destroy_restaurant_path("1")
+      expect(page).to have_content("Log in")
+    end
+  end
+
+  context 'As a signed in user viewing a restaurant I did not created' do
+    before :each do
+      restaurant = FactoryGirl.create(:restaurant)
+      other_user = FactoryGirl.create(:user)
+      sign_in_as other_user
+    end
+
+    scenario 'I cannot see edit or delete links on restaurant page'
+      visit restaurant_path(restaurant)
+      expect(page).to not_have_content("Edit Restaurant")
+      expect(page).to not_have_content("Delete Restaurant")
+    end
```

Comment:
This is similar to how your tests above should look (about not expecting content).